flow/project.py

```diff
@@ -747,32 +877,34 @@ def np_callable(job):
         directives.setdefault('omp_num_threads', 0)
         directives.setdefault('processor_fraction', 1)
 
-        # Evaluate strings and callables for job:
+        # Evaluate strings and callables for jobs:
         def evaluate(value):
             if value and callable(value):
-                return value(job)
+                return value(*jobs)
             elif isinstance(value, str):
-                return value.format(job=job)
+                return value.format(*jobs)
             else:
                 return value
         return {key: evaluate(value) for key, value in directives.items()}
 
-    def _submit_cmd(self, entrypoint, ignore_conditions, job=None):
-        entrypoint = self._determine_entrypoint(entrypoint, dict(), job)
+    def _submit_cmd(self, entrypoint, ignore_conditions, jobs=None):
+        entrypoint = self._determine_entrypoint(entrypoint, dict(), jobs)
         cmd = "{} run -o {}".format(entrypoint, self.name)
-        cmd = cmd if job is None else cmd + ' -j {}'.format(job)
+        cmd = cmd if jobs is None else cmd + ' -j {}'.format(' '.join(map(str, jobs)))
         cmd = cmd if self.options is None else cmd + ' ' + self.options
         if ignore_conditions != IgnoreConditions.NONE:
             return cmd.strip() + ' --ignore-conditions=' + str(ignore_conditions)
         else:
             return cmd.strip()
 
-    def _run_cmd(self, entrypoint, operation_name, operation, directives, job):
+    def _run_cmd(self, entrypoint, operation_name, operation, directives, jobs):
         if isinstance(operation, FlowCmdOperation):
-            return operation(job).lstrip()
+            return operation(*jobs).lstrip()
         else:
-            entrypoint = self._determine_entrypoint(entrypoint, directives, job)
-            return '{} exec {} {}'.format(entrypoint, operation_name, job).lstrip()
+            entrypoint = self._determine_entrypoint(entrypoint, directives, jobs)
+            return '{} exec {} {}'.format(
+                        entrypoint, operation_name, ' '.join(map(str, jobs))
+                    ).lstrip()
```

Comment:
Please use f-strings.

Reply:
You marked this conversation as resolved, yet this is still not an f-string?

Reply:
@csadorf I apologize. I wrote a comment but instead of posting, I must have clicked resolved. I'll take care of this from the next time.
The comment was: 
Throughout the code-base, we don't use f-strings. Instead, we format the strings in this manner `'{name}-{operation}'.format(name=name, operation=op_name)`.
I understand that using f-strings will be more efficient here but I think we should be consistent.
If you want, I'll use an f-string in this PR or I will make another PR which will internally convert all the string formatting to f-strings formatting to maintain consistency.
Please suggest

Reply:
I think it is totally fine to prefer the use of f-strings moving forward even if we use `"...".format()` in other sections of the code.

Reply:
Will make the changes now.